.gitignore

```diff
@@ -4,3 +4,4 @@ Propel/map
 Propel/om
 
 Tests/app/tmp/
+.idea
```

Comment:
You should not add this to .gitignore

See https://insight.symfony.com/what-we-analyse/git.user_specific_ignored_file